From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 16 Sep 2017 23:43:29 +0200
Subject: Remove external images

Due to privacy concerns these are removed from the documentation in
Debian.
See also lintian tag privacy-breach-generic.

Upstream includes images from external sites to show thirdparies
screenshots.
Replace them with a link instead so the user can choose to not browse
the image for privacy reasons.

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/applications.rst | 4 +---
 docs/donate.rst       | 5 -----
 docs/thirdparty.rst   | 5 +----
 3 files changed, 2 insertions(+), 12 deletions(-)

diff --git a/docs/applications.rst b/docs/applications.rst
index d69bb19..14908c3 100644
--- a/docs/applications.rst
+++ b/docs/applications.rst
@@ -8,14 +8,12 @@ Streamlink Applications
 Streamlink Twitch GUI
 ---------------------
 
-.. image:: https://user-images.githubusercontent.com/467294/199009198-011d23f2-a884-4bf8-94d0-bb304ceecc08.jpg
-    :alt: Streamlink Twitch GUI
-
 :Description: A multi platform Twitch.tv browser for Streamlink
 :Type: Graphical User Interface
 :OS: |Windows| |MacOS| |Linux|
 :Author: `Sebastian Meyer <https://github.com/bastimeyer>`_
 :Website: https://streamlink.github.io/streamlink-twitch-gui
+:Screenshot: https://user-images.githubusercontent.com/467294/199009198-011d23f2-a884-4bf8-94d0-bb304ceecc08.jpg
 :Source code: https://github.com/streamlink/streamlink-twitch-gui
 :Info: An NW.js based desktop web application. Browse Twitch.tv and watch multiple streams at once.
     Filter streams by language, receive desktop notifications when followed channels start
diff --git a/docs/donate.rst b/docs/donate.rst
index 111c3b3..64441a2 100644
--- a/docs/donate.rst
+++ b/docs/donate.rst
@@ -26,21 +26,18 @@ Team members
 
     .. grid-item-card::
         :padding: 3
-        :img-top: https://github.com/bastimeyer.png?size=100
         :link: https://github.com/bastimeyer
         :link-alt: @bastimeyer
         :text-align: center
 
     .. grid-item-card::
         :padding: 3
-        :img-top: https://github.com/gravyboat.png?size=100
         :link: https://github.com/gravyboat
         :link-alt: @gravyboat
         :text-align: center
 
     .. grid-item-card::
         :padding: 3
-        :img-top: https://github.com/mkbloke.png?size=100
         :link: https://github.com/mkbloke
         :link-alt: @mkbloke
         :text-align: center
@@ -55,14 +52,12 @@ Inactive / former
 
     .. grid-item-card::
         :padding: 3
-        :img-top: https://github.com/back-to.png?size=100
         :link: https://github.com/back-to
         :link-alt: @back-to
         :text-align: center
 
     .. grid-item-card::
         :padding: 3
-        :img-top: https://github.com/beardypig.png?size=100
         :link: https://github.com/beardypig
         :link-alt: @beardypig
         :text-align: center
diff --git a/docs/thirdparty.rst b/docs/thirdparty.rst
index de2f5f5..a432bff 100644
--- a/docs/thirdparty.rst
+++ b/docs/thirdparty.rst
@@ -55,15 +55,12 @@ LiveProxy
 Wtwitch
 -------
 
-.. image:: https://user-images.githubusercontent.com/25400030/97115072-6951e600-16ec-11eb-88f7-51939c0d0bc1.jpg
-  :align: center
-  :alt: Wtwitch screenshot of checking subscriptions
-
 :Description: Browse Twitch streams; subscribe to streamers locally without an account
 :Type: TUI (terminal user interface)
 :OS: |Linux| |MacOS|
 :Author: `Hunter Peavey <https://krathalan.net>`_
 :Website: https://git.sr.ht/~krathalan/wtwitch
+:Screenshot: https://user-images.githubusercontent.com/25400030/97115072-6951e600-16ec-11eb-88f7-51939c0d0bc1.jpg
 :Info: wtwitch is a Bash program that lets you browse the top games on Twitch, the |br|
   top streamers for a given game, and lets you check the status of streamers you |br|
   subscribe to. Open streams easily with settings that save your preferred quality |br|
